masterlist.yaml

```diff
@@ -1327,3 +1327,21 @@ plugins:
   - name: 'Convenient Horses.esp'
     after:
       - 'Open Cities Skyrim.esp'
+  - name: 'TheChoiceIsYours.esp'
+    after:
+      - 'BetterQuestObjectives.esp'
+      - 'Book Covers Skyrim.esp'
+      - 'Immersive Citizens - AI Overhaul.esp'
+      - 'Relationship Dialogue Overhaul.esp'
```

Comment:
No major conflicts, but these mods will overwrite changes made by The choice is yours.